addons/analytic/models/analytic_plan.py

```diff
@@ -145,17 +150,21 @@ def _compute_analytic_account_count(self):
         for plan in self:
             plan.account_count = len(plan.account_ids)
 
+    def _compute_all_children_ids(self):
+        for plan in self:
+            plan.all_children_ids = plan.children_ids + plan.children_ids.all_children_ids
+
     @api.depends('account_ids', 'children_ids')
     def _compute_all_analytic_account_count(self):
         plans_count = dict(
             self.env['account.analytic.account']._read_group(
-                domain=[('root_plan_id', 'in', self.ids)],
+                domain=[('plan_id', 'child_of', self.ids)],
                 aggregates=['id:count'],
-                groupby=['root_plan_id']
+                groupby=['plan_id']
             )
         )
         for plan in self:
-            plan.all_account_count = plans_count.get(plan, 0)
+            plan.all_account_count = sum(plans_count.get(planc, 0) for planc in (plan.all_children_ids + plan))\
```

Comment:
Should probably not use `planc` as it does not mean anything. Maybe only `p` or something else if you think about something beautiful :smile: .
And remove the `\` at the end